packages/manager/apps/zimbra/src/pages/onboarding/index.tsx

```diff
@@ -1,43 +1,109 @@
-import React from 'react';
+import React, { useContext, useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
-import { OnboardingLayout } from '@ovh-ux/manager-react-components';
+import { Card, OnboardingLayout } from '@ovh-ux/manager-react-components';
 import {
   ButtonType,
   PageLocation,
   useOvhTracking,
+  ShellContext,
 } from '@ovh-ux/manager-react-shell-client';
+import { getOrderURL } from '@ovh-ux/manager-module-order';
+import {
+  ONBOARDING_ORDER_CTA,
+  GO_TO,
+  GUIDE_HOW_TO_CONFIGURE,
+  GUIDE_ADMINISTRATOR,
+  GUIDE_WEBMAIL,
+} from '@/tracking.constant';
 import onboardingImgSrc from './onboarding-img.png';
-import { JOIN_ZIMBRA_BETA } from '@/tracking.constant';
+import {
+  ZIMBRA_ADMINISTRATOR_GUIDE,
+  ZIMBRA_HOW_TO_CONFIGURE_GUIDE,
+  ZIMBRA_WEBMAIL_GUIDE,
+} from '@/guides.constants';
+import { ORDER_LINK, WEBSITE_LINK } from './onboarding.constants';
 
 export default function Onboarding() {
   const { t } = useTranslation('onboarding');
+  const context = useContext(ShellContext);
+  const { ovhSubsidiary } = context.environment.getUser();
+  const region = context.environment.getRegion();
+  const orderBaseURL = getOrderURL('orderDomain', region, ovhSubsidiary);
 
-  const title: string = t('title');
-  const description: string = t('description');
   const { trackClick } = useOvhTracking();
 
   const onOrderButtonClick = () => {
     trackClick({
       location: PageLocation.page,
       buttonType: ButtonType.button,
       actionType: 'navigation',
-      actions: [JOIN_ZIMBRA_BETA],
+      actions: [ONBOARDING_ORDER_CTA],
     });
 
-    window.open(
-      'https://labs.ovhcloud.com/en/zimbra-beta/',
-      '_blank',
-      'noopener',
-    );
+    window.open(`${orderBaseURL}${ORDER_LINK}`, '_blank', 'noopener');
   };
 
+  const onboardingGuides = [
+    {
+      key: 1,
+      href: useMemo(() => {
+        return (
+          ZIMBRA_ADMINISTRATOR_GUIDE[ovhSubsidiary] ||
+          ZIMBRA_ADMINISTRATOR_GUIDE.DEFAULT
+        );
+      }, [ZIMBRA_ADMINISTRATOR_GUIDE, ovhSubsidiary]),
+      tracking: GUIDE_ADMINISTRATOR,
+    },
+    {
+      key: 2,
+      href: useMemo(() => {
+        return (
+          ZIMBRA_HOW_TO_CONFIGURE_GUIDE[ovhSubsidiary] ||
+          ZIMBRA_HOW_TO_CONFIGURE_GUIDE.DEFAULT
+        );
+      }, [ZIMBRA_HOW_TO_CONFIGURE_GUIDE, ovhSubsidiary]),
+      tracking: GUIDE_HOW_TO_CONFIGURE,
+    },
+    {
+      key: 3,
+      href: useMemo(() => {
+        return (
+          ZIMBRA_WEBMAIL_GUIDE[ovhSubsidiary] || ZIMBRA_WEBMAIL_GUIDE.DEFAULT
+        );
+      }, [ZIMBRA_WEBMAIL_GUIDE, ovhSubsidiary]),
+      tracking: GUIDE_WEBMAIL,
+    },
+  ];
+
   return (
     <OnboardingLayout
-      title={title}
+      title={t('title')}
       img={{ src: onboardingImgSrc }}
-      description={description}
+      description={t('description')}
```

Comment:
we need to pass an `OdsText` instead of a string now to match the required design because of some changes that happened in `OnboardingLayout`